developer/src/kmc-package/test/test-package-compiler.ts

```diff
@@ -157,11 +159,18 @@ describe('KmpCompiler', function () {
     const kmpCompiler = new KmpCompiler(callbacks);
 
     let kmpJson = kmpCompiler.transformKpsToKmpObject(kpsPath);
+    if(kmpJson && callbacks.messages.length == 0) {
+      const validator = new PackageValidation(callbacks);
+      validator.validate(kmpJson); // we'll ignore return value and rely on the messages
+    }
+
     if(kmpJson && callbacks.messages.length == 0) {
       // We'll try building the package if we have not yet received any messages
       kmpCompiler.buildKmpFile(kpsPath, kmpJson)
```

Comment:
Should `kmpCompiler.buildKmpFile` depend on `validator.validate(kmpJson)` returning true?

Reply:
not in this test -- we ignore the return value on purpose because we are verifying that certain messages are generated in the build process.